test/tentacles/core_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns tentacles.core-test
   (:use clojure.test)
-  (:require [tentacles.core :as core]))
+  (:require [tentacles.core :as core]
+            [clj-http.core :as http]))
```

Comment:
Seems like this was added but is unused?

Reply:
good eye, I'll remove it!